### DISCUSSION TITLE
Increase number of validators

### DISCUSSION THREAD

**Discussion by bunghi** [oasisprotocol/community-forum#4] [Mainnet]
Hi,

I'd like to propose the max_validators=80 of consensus group to be inreased to 200 and still based on escrow. It is really difficult for new validators to join right now, you need at least 100K $ROSE to have a place somewhere over 70th, when minimum escrow required is 100 ROSE (useless).

Thanks

**Comment by expyri** [comment]
right !

**Comment by tjanez** [comment]
The number of validators was increased to 100 in the [Cobalt Upgrade](https://github.com/oasisprotocol/community/discussions/18).

**Reply by JODIN1727** [reply]
Yes.. I agree

**Comment by tjanez** [comment]
The number of validators has been increased to 110 in the [2021-08-31 Parameter Upgrade](https://docs.oasis.dev/general/run-a-node/upgrade-log#2021-08-31-16-00-utc-parameter-update).
